### DISCUSSION TITLE
What is the difference between FlowSOM_cluster and FlowSOM_metacluster?

### DISCUSSION THREAD

**Discussion by denvercal1234GitHub** [ImmuneDynamics/Spectre#93] [Q&A] [answered]
Hi there, 

Thank you again for this well-constructed and explained package. 

But, what is the difference between FlowSOM_cluster and FlowSOM_metacluster? And, when to use which one? 😅

**Comment by tomashhurst** [comment, marked as answer]
@denvercal1234GitHub excellent question! Essentially FlowSOM creates the _clusters_ initially, and then groups them together into _metaclusters_. There are a number of reasons for this, but fundamentally it is about providing levels of resolution -- the clusters are far more granular and are finding subtle changes in data structure, metaclusters will (hopefully) collect those into groups that more closely resemble 'populations'. 

I would focus mainly on the metaclusters, where the clusters can be used in some situations to dive deeper where required.

**Reply by SamGG** [reply]
Just to increment Thomas's answer, you have to take care of the ratio between the number of clusters to the number of meta-clusters. 30 or 40 meta-clusters might be the upper limit for 100 clusters. If you want more meta-clusters, you have to increase the number of meta-clusters : for example a grid of 14x14 = 196 clusters for 60 meta-clusters. This still depends on the dataset and trials...

**Comment by denvercal1234GitHub** [comment]
Thank you! 😊
